server/notification-providers/46elks.js

```diff
@@ -0,0 +1,52 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class Elks extends NotificationProvider {
+    name = "Elks";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+
+        try {
+
+            // API credentials
+            const username = notification.elksUsername;
+            const password = notification.elksAuthToken;
+            const authKey = Buffer.from(username + ":" + password).toString("base64");
+
+            // Set the SMS endpoint
+            const url = "https://api.46elks.com/a1/sms";
+
+            // Request data object
+            let data = {
+                from: notification.elksFromNumber,
+                to: notification.elksToNumber,
+                message: msg
+            };
+
+            data = new URLSearchParams(data);
```

Comment:
Please use `UrlSearchParams` like the other notification providers

Example:

https://github.com/louislam/uptime-kuma/blob/bbc75b840bc0eefa1d7296a78487ad8ed4c8a97a/server/notification-providers/gtx-messaging.js#L17-L24